### PR TITLE
Fix collins-notify for ruby 2.2.0

### PR DESCRIPTION
**Pull request by byxorna**
@maddalab @Primer42 @roymarantz @defect This fixes the `#respond_to?` invocations to actually find protected methods, which is preventing collins-notify from working on newer rubies. We ask respond_to to search all methods, not just public ones.

[0] http://ruby-doc.org/core-1.9.3/Object.html#method-i-respond_to-3F
[1] http://ruby-doc.org/core-2.2.2/Object.html#method-i-respond_to-3F

I have tested this and confirmed functionality in both 2.2.0 and 1.9.2.

### CODE REVIEW

File: support/ruby/collins-notify/collins_notify.gemspec [hunk 1]
```diff
@@ -44,28 +44,10 @@ Gem::Specification.new do |s|
...
-    s.add_dependency(%q<collins_client>, ["~> 0.2.10"])
-    s.add_dependency(%q<hipchat>, ["~> 0.7.0"])
-    s.add_dependency(%q<mail>, ["~> 2.4.4"])
-    s.add_dependency(%q<nokogiri>, ["~> 1.5.2"])
-  end
+  s.add_runtime_dependency('addressable', '~> 2.3.2')
+  s.add_runtime_dependency('collins_client', '~> 0.2.10')
+  s.add_runtime_dependency('hipchat', '~> 0.7.0')
+  s.add_runtime_dependency('mail', '~> 2.4.4')
+  s.add_runtime_dependency('nokogiri', '~> 1.5.2')
```
Comment (defect) [inline support/ruby/collins-notify/collins_notify.gemspec]: Maybe move the dev dependencies in here too and just point the Gemfile to the gemspec?

Comment (byxorna) [inline support/ruby/collins-notify/collins_notify.gemspec]: fixed.

**Comment by defect** [comment]
lgtm

**Comment by roymarantz** [comment]
:+1:

**Pull request merged**